operations/structure.md

```diff
@@ -0,0 +1,45 @@
+# Structure and responsibilities
+
+This page describes the structure and roles of our Product Team.
+
+## Responsibilities
+
+2i2c's Operatoins team is responsible for defining and overseeing our systems for work coordination, planning, and accountability.
+
+Below are major areas of responsibility.
+
+### Define systems and processes for coordination
+
+- Define our meeting and coordination systems for planning our work.
+- Define systems for sharing information between team members.
+
+### Refine work items to be actionable and clear
+
+- Translate strategic priorities and deliverables into actionable work items.
+- Ensure that our work items have a clear scope and path forward for completion.
+- Communicate with our Product and Engineering teams to align our teams on a plan.
+
+### Oversee work for projects and teams
+
+- Assign work items to team members on a given project according to team priorities.
+- Hold team members accountable for completing work on-time, or for adjusting scope as needed.
+
+### Communicate progress and status to key stakeholders
+
+- Make our work transparent, discoverable, and understanding to internal and external stakeholders.
+- Communicate with internal and external stakeholders to share information about the status and progress of major work items.
```

Comment:
I am not sure Operations should directly connect to the external stakeholders. Communications with external, I think, should happen through Community and Partnership areas.

Reply:
I was thinking about a project manager style role, and how they are often communicating the status of projects, writing posts about it, etc. For example - the communication you're having with Arnim around the GESIS work. Maybe we should make it clear that this is when there are projects with external collaborators?

Reply:
Again, in an ideal world, the PM gets input from the community/prod side about the external request. In the interim, for instance, it is ok for me to have conversations with Arnim, but in the future, that conversation should regularly happen between the external collaborator and (Comm and Prod), IMHO. The PM should not be tasked to inform external parties, their focus is to push forward the project accordingly to directions from the external-facing team.